src/apps/backend/modules/access-token/rest-api/access-token-router.ts

```diff
@@ -1,14 +1,11 @@
-/* eslint-disable @typescript-eslint/no-misused-promises */
-import { Router } from 'express';
+import { ApplicationRouter } from '../../application';
 
-import AccessTokenController from './access-token-controller';
+import * as AccessTokenController from './access-token-controller';
 
-export default class AccessTokenRouter {
-  public static getRoutes(): Router {
-    const router = Router();
+export default class AccessTokenRouter extends ApplicationRouter {
+  configure(): void {
+    const { router } = this;
 
     router.post('/', AccessTokenController.createAccessToken);
```

Comment:
just for some added safety, would it be possible to configure a list of domains from which we allow access? 

Seen this being done - 
<img width="1194" alt="image" src="https://github.com/jalantechnologies/boilerplate-mern/assets/92290744/44ba56f5-1971-4666-ae28-20f1a621487c">

Reply:
was pointing this way only our frontend clients (web and mobile) can access the backend apis